lib/client/config_e2e_test.go

```diff
@@ -89,6 +89,89 @@ var _ = testutils.E2eDatastoreDescribe("with config option API tests", testutils
 		Expect(cs).To(Equal(client.ConfigLocationGlobal))
 	})
 
+	It("should handle node IP in IP tunnel address", func() {
```

Comment:
Yup - and that is why I didn't just add node-specific Felix config in this PR.  It's not required for KDD routing, and needs a little more thought regarding interactions with this field.